internal/gensupport/retry.go

```diff
@@ -26,6 +26,8 @@ type Backoff interface {
 var (
 	// Default per-chunk deadline for resumable uploads.
 	defaultRetryDeadline = 32 * time.Second
+	// Default per-chunk transfer timeout for resumable uploads.
+	defaultTransferTimeout = 0 * time.Second
```

Comment:
I think we don't need a variable for this given that zero is the language default for time.Duration.

Reply:
I kept it this way to make it easier to update the default value from 0 to something else in the future.  Anyways updated the changes as you said.